package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -93,10 +93,7 @@ If you use the Path Similarity Analysis (PSA) code in
 If you use the implementation of the ENCORE ensemble analysis in
 :mod:`MDAnalysis.analysis.encore` please cite [Tiberti2015]_.
 
-.. [Tiberti2015] M. Tiberti, E. Papaleo, T. Bengtsen, W. Boomsma,
-   and K. Lindorff-Larsen. ENCORE: Software for quantitative ensemble
-   comparison. *PLoS Comput Biol*, **11** (2015), e1004415.  doi:
-   `10.1371/journal.pcbi.1004415`_
+[Tiberti2015]_
```

Comment:
So here I have a fundamental question. Sphinx doesn't like that we define the reference `[Tiberti2015]` in several places. So far in my clean up here my strategy was to use one definition in the source code and use that link in the rst files. Is that OK. Should we go a different route?